dev/src/components/__tests__/isValid.test.tsx

```diff
@@ -0,0 +1,54 @@
+import {isValid} from "../../Helpers"
+import {User, EnrollmentLink, Enrollment, Submission, SubmissionLink} from "../../../proto/ag/ag_pb";
+
+describe("User and enrollment validation", ()=> {
+    it("User should be valid", () =>{
+        const user = new User().setId(1).setName("Test User").setEmail("mail@mail.com").setStudentid("1234567")
+        const isValidUser = isValid(user)
+        expect(isValidUser).toBe(true)
+
+    });
+    it("User should not be valid if name is empty", () =>{
+        const user2 = new User().setId(2).setEmail("mail@mail.com").setStudentid("1234567")
+        const isValidUser = isValid(user2)
+        expect(isValidUser).toBe(false)
+        
+    });
+    
+    //Should isValid have a function that checks that it is a legit email, and not just a string with length > 0?
```

Comment:
Yes, we should have / use such a function if we don't already have an email checker... Didn't check. Please feel free to add tests for this; I think maybe there exists a library for this.